qemu/tests/cgroup.py

```diff
@@ -2059,6 +2062,57 @@ def memory_move():
         else:
             return (out)
 
+    def cfs_bandwidth():
+        """
+        CFS bandwidth limit
+        """
+        def stress_thread():
+            session.cmd(stress_cmd, 120)
+
+        logging.info("Setup cgroup subsystem: cpu")
+        modules = CgroupModules()
+        if (modules.init(['cpu']) != 1):
+            test.fail("Can't mount cpu cgroup modules")
+        cgroup = Cgroup('cpu', '')
+        cgroup.initialize(modules)
+
+        # Create VM with smp=1
+        params['smp'] = 1
+        params["vcpu_sockets"] = 1
+        vm_name = params["main_vm"]
+        params["start_vm"] = "yes"
+        env_process.preprocess_vm(test, params, env, vm_name)
+        vm = env.get_vm(params["main_vm"])
+        login_timeout = int(params.get("login_timeout", 360))
+        session = vm.wait_for_login(timeout=login_timeout)
+
+        cgroup.mk_cgroup()
+        cgroup.set_property("cpu.cfs_period_us", 100000, 0)
+        cgroup.set_property("cpu.cfs_quota_us", 50000, 0)
+        assign_vm_into_cgroup(vm, cgroup, 0)
+
+        check_install_stress = params["check_install_stress"]
+        if session.cmd_status(check_install_stress):
```

Comment:
I guess here it should be `if session.cmd_status(check_install_stress) != 0:`

Reply:
`if 0` is equivalent to `if False`

Reply:
@hereischen Yes, but `!=0` does not equal to `True`. I am not sure, but suppose that command status returns `-1`, will that lead error?

Reply:
@hereischen Well, it works right, ignore me then. Thanks.